frontend-public/src/components/authentication/Authentication.js

```diff
@@ -24,65 +24,70 @@ const defaultProps = {
   userInfo: {},
 };
 
-export const Authentication = (props) => {
-  const hamburgerMenu = (
-    <Menu>
-      <Menu.Item>
-        <Button type="tertiary">
-          <Link to={route.DASHBOARD.route}>My Mines</Link>
-        </Button>
-      </Menu.Item>
-      <Divider style={{ margin: "0" }} />
-      <Menu.Item>
-        <Button type="tertiary" onClick={signOutFromSiteMinder}>
-          Log out
-        </Button>
-      </Menu.Item>
-    </Menu>
-  );
+export class Authentication extends Component {
+  handleLogout = () => {
```

Comment:
What are these changes for? Do they relate to the 404 somehow or to something else?

Reply:
This relates to the second part of my PR, " - Added a possible fix to the mineSpace toast message bug"

Reply:
Oh, glorious! Marking as resolved